crates/anvil/src/eth/error.rs

```diff
@@ -278,7 +266,7 @@ pub(crate) fn decode_revert_reason(out: impl AsRef<[u8]>) -> Option<String> {
     if out.len() < SELECTOR_LEN {
         return None
     }
-    String::decode(&out[SELECTOR_LEN..]).ok()
+    DynSolType::String.abi_decode(&out[SELECTOR_LEN..]).ok().and_then(|v| v.as_str().map(|s| s.to_owned()))
```

Comment:
@DaniPopes is this the best way to get the Option<String>?

Reply:
No, but just use foundry_common decode_revert https://github.com/foundry-rs/foundry/blob/b51c05734d0bc8879df64d00f536acb7413c84fa/crates/evm/core/src/decode.rs#L30C8-L30C21

Reply:
It would be String::abi_decode anyway.